sample/src/main/assets/rule/simple-rules.yml

```diff
@@ -0,0 +1,6 @@
+name: step1_larvae_3_total
+description: larvae 3 total
+priority: 1
+condition: "step1_larvae_total == 1 && step1_larvae_1_total == 1"
```

Comment:
`step1_larvae_1_total` only this condition is being respected.

If `step1_larvae_total` is blank but `step1_larvae_1_total == 1` is true, `step1_larvae_3_total` still shows

Reply:
The above can be reproduced with 2 dips and using `dip2`.

Additionally, the relevance is not being respected for `dip1`

Reply:
If you start by filling in `step1_larvae_1_total` with `1`, then fill in `step1_larvae_total` with `1`, relevance is not fired.

Reply:
> `step1_larvae_1_total` only this condition is being respected.
> 
> If `step1_larvae_total` is blank but `step1_larvae_1_total == 1` is true, `step1_larvae_3_total` still shows

You can first fill in both fields with `1` then delete `step1_larvae_total`, field doesn't disappear. Then delete `step1_larvae_1_total`, field disappears, and if you re-type 1 in `step1_larvae_1_total`, it re-appears

Reply:
> `step1_larvae_1_total` only this condition is being respected.
> 
> If `step1_larvae_total` is blank but `step1_larvae_1_total == 1` is true, `step1_larvae_3_total` still shows

This occurs due to the required property, when its empty it validates the 'emptiness' and leaves it there. When you remove the required part it works.

Reply:
> > `step1_larvae_1_total` only this condition is being respected.
> > If `step1_larvae_total` is blank but `step1_larvae_1_total == 1` is true, `step1_larvae_3_total` still shows
> 
> You can first fill in both fields with `1` then delete `step1_larvae_total`, `step1_larvae_3_total` field doesn't disappear. Then delete `step1_larvae_1_total`, `step1_larvae_3_total` field disappears, and if you re-type 1 in `step1_larvae_1_total`, it re-appears

This occurs due to the required property, when its empty it validates the 'emptiness' and leaves it there. When you remove the required part it works.

Reply:
Ok, could you create an issue with these observations, so that we have it documented and maybe later worked on to have more predictable behavior?

Reply:
https://github.com/OpenSRP/opensrp-client-native-form/issues/328